clients/dpd-client/build.rs

```diff
@@ -92,7 +93,61 @@ fn main() -> Result<()> {
                     slog::debug!(state.log, "client response"; "result" => ?result);
                 }
             })
-	    .with_derive("PartialEq")
+            .with_derive("PartialEq")
+            // NOTE: This should all go away when we can open-source the real
+            // `dpd-client` from the repo itself. These are used to ensure that
+            // we pick up things like the display implementations.
+            .with_replacement(
+                "ActiveCableMediaInterfaceId",
+                "transceiver_decode::ActiveCableMediaInterfaceId",
+                std::iter::once(TypeSpaceImpl::Display)
```

Comment:
Does this work?
```suggestion
                &[TypeSpaceImpl::Display]
```
or without the `&` perhaps?

Reply:
You need to do something like `[TypeSpaceImpl::Display].into_iter()`, so I've opted to keep this as-is.